src/subscription_manager/gui/about.py

```diff
@@ -34,6 +34,7 @@
             "Red Hat trademarks are not licensed under GPLv2. No permission is "
             "granted to use or replicate Red Hat trademarks that are incorporated "
             "in this software or its documentation.\n")
+AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
```

Comment:
relpaths can also be kind of weird, especially when running as root.

```
[~]$ cd
[~]$ mkdir docs
[~]$ ln -s /etc/shadow- docs/authors.txt
[~]$ sudo subscription-manager-gui 
```

![sub-man-abouts](https://cloud.githubusercontent.com/assets/15162/9742118/58093ec4-562c-11e5-85c2-bb50464a1799.png)